rest/handler/tracinghandler.go

```diff
@@ -7,18 +7,42 @@ import (
 	"github.com/zeromicro/go-zero/core/lang"
 	"github.com/zeromicro/go-zero/core/trace"
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/propagation"
 	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
 	oteltrace "go.opentelemetry.io/otel/trace"
 )
 
+const (
+	traceKeyStatusCode = "http.status_code"
+)
+
 var notTracingSpans sync.Map
 
 // DontTraceSpan disable tracing for the specified span name.
 func DontTraceSpan(spanName string) {
 	notTracingSpans.Store(spanName, lang.Placeholder)
 }
 
+type traceResponseWriter struct {
```

Comment:
How is the logic with Hijack/Flush method?

Please check it out in `loghandler.go`.

Reply:
Added. Thanks for your review.